cmd/wio/wio.go

```diff
@@ -101,6 +101,10 @@ var runFlags = []cli.Flag{
         Name:  "port",
         Usage: "Specify upload port",
     },
+	cli.StringFlag{
```

Comment:
I like this but it will be more intuitive for a user to just do `wio run arg1 arg2 arg3 arg4`. We have to figure out when targetName is provided to run. Maybe check the first argument and see if it is a valid target and if not use all the args as arguments. Idk we can think about this

Reply:
But i guess `args` ain't that bad to type either so idk lol

Reply:
yeah but currently it just takes everything as a target (even though you can only run one target...)

it's probably fine for now, really just need it for testing